trivup/trivup.py

```diff
@@ -426,8 +426,10 @@ def create_file_from_template(self, relpath, unique=False,
 
     def resource_path(self, relpath):
         """ @returns the full path to an application class resource file """
-        return pkg_resources.resource_filename('trivup',
-                                               os.path.join('apps', self.__class__.__name__, relpath))  # noqa: E501
+        return importlib.resources.as_file(
+            importlib.resources.files('trivup') / 'apps' /
+                self.__class__.__name__ / relpath  # noqa: E131
+        )
```

Comment:
`importlib.resources.as_file` is not present in Python 3.8 which we are using in python client for all the builds and wheel generation right now. I can check to upgrade that to 3.8+ something but will take some time to test as well.

For fixing in Python 3.8, you can use `importlib_resources` instead. [Ref](https://github.com/nomic-ai/gpt4all/issues/1605#issuecomment-1804757403)

Reply:
Ok, closing this as you upgraded Py version